microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -552,6 +637,54 @@ The functions tag is not unique to your installation of Microsoft Edge. The tag
 
 
+<!-- ====================================================================== -->
+## Rewards
+
+Microsoft Rewards (the "Program") enables you to earn redeemable points for activities such as qualified searches, acquisitions, and other offers from Microsoft. These points can then be redeemed for gift cards or non-profit donations or automatic contributions to subscriptions. Microsoft Edge has built-in support for all existing avenues for earning points, and also introduces new and exclusive means to earn more points. You can track your points, and you can stay up-to-date with limited period offers to get the most out of the program.
+
+For non-members, Microsoft Edge presents personalized joining offers based on the page you're on. Once you engage and choose to enroll, Microsoft Edge uses your Microsoft account identity to sign you up and start the rewards service.  The rewards service automatically adds points to your account for searching, playing, and shopping on Microsoft Edge. Microsoft Edge doesn't send your personal information or browsing data to any third-party services.
+
+Microsoft Edge periodically offers custom browsing streak offers, for browsing on Microsoft Edge for a few minutes every day for a limited period. You can track your streak progress; points earned and discover new offers on the [Rewards dashboard](https://rewards.bing.com/) or in the **Action center** in the Microsoft Edge sidebar.
+
+The Rewards dashboard (`rewards.bing.com`):
+
+![Rewards Dashboard UI](./index-images/rewardsdashboard.png)
+
+The **Action center** in the Microsoft Edge sidebar:
+
+![Rewards Sidebar UI](./index-images/rewardssidepane.png)
+
+Rewards is an opt-in program. To turn off all Rewards experiences and data sharing, go to `edge://settings/profiles/rewards`, and then in the **Microsoft Rewards** section, turn off the **Earn Microsoft Rewards in Microsoft Edge** toggle:
+
+![Rewards Settings UI](./index-images/rewardssettings.png)
+
+Offers may vary by market. 
+
+#### How to earn
+
+Microsoft Edge provides the greatest number of avenues to earn rewards points while browsing the web.
+
+A _search_ is the act of an individual user manually entering text for the good faith purpose of obtaining Bing search results for the user's own research purposes, and does not include any query that's entered by a bot, macro, or other automated or fraudulent means of any kind ("Search").
+
+An _acquisition_ is the process of purchasing goods or downloading and acquiring a license for digital content from Microsoft, whether free or paid ("Acquisition"). Rewards points are not offered for every purchase from Microsoft. Microsoft may offer additional opportunities to earn points from time to time, and each point-earning offer will not be available in perpetuity.
+
+#### Device requirements
```

Comment:
@majenny - as part of Review process, pls test the links in remainder of this section.
The links assume you're signed in, which I'm not (I should enter a comment to drive that). Ihab helped set up my previous machine to partly use Rewards UI & a test account; that involved several locations, so I'm not attempting that level of testing on this machine.